google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -1114,6 +1114,40 @@ public boolean deleteDefaultAcl(Entity entity) {
     return storage.deleteDefaultAcl(getName(), entity);
   }
 
+  /**
+   * Delete lifecycle rules of the requested bucket.
```

Comment:
Let's change this to delete the lifecycle rules of *this* bucket

Reply:
done